packages/toolpad-components/src/Autocomplete.tsx

```diff
@@ -91,10 +92,14 @@ function Autocomplete({
     if (!value) {
       setSelectedVal(null);
     }
-  }, [value]);
+    if (defaultValue) {
+      setSelectedVal(defaultValue);
+    }
+  }, [value, defaultValue]);
 
   return renderFormInput(
     <MuiAutocomplete
+      defaultValue={defaultValue}
```

Comment:
Please, take a look at the [Select](https://github.com/mui/mui-toolpad/blob/5c6f80454d674fb91c6ed6e7864ca6e9e046fc14/packages/toolpad-components/src/Select.tsx#L123) component on how `defaultValue` is implemented. It must work the same way for Autocomplete